Chat App/Chat App/Extensions/TableViewCell+Extensions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  TableViewCell+Extensions.swift
+//  Chat App
+//
+//  Created by tornike on 15.05.23.
+//
+
+import Foundation
+
+extension Date {
+        func formatDate() -> String {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "HH:mm"
+        return formatter.string(from: self)
+    }
+}
```

Comment:
Something happened to tabulation here. 

Also, please make `dateFormat` an input value for this function, this way function will be much more reusable.

Reply:
you mean like this? extension Date {
    func formatDate(dateFormat: String) -> String {
        let formatter = DateFormatter()
        formatter.dateFormat = dateFormat
        return formatter.string(from: self)
    }
}

Reply:
Yeap, absolutely) 

This way you will be able to call it from different classes and get strings of different formats if you need.